test/functional/cmdLineTests/cmdLineTest_J9tests/playlist.xml

```diff
@@ -62,4 +62,21 @@
 			<subset>9+</subset>
 		</subsets>
 	</test>
+	
+	<test>
+		<testCaseName>cmdLineTest_J9test_extended</testCaseName>
+		<command>$(JAVA_COMMAND) -Xdump $(JVM_OPTIONS) -DFIBJAR=$(Q)$(JVM_TEST_ROOT)$(D)functional$(D)cmdLineTests$(D)utils$(D)utils.jar$(Q) \
+	-DTESTDIR=$(Q)$(TEST_RESROOT)$(Q) -DRESJAR=$(CMDLINETESTER_RESJAR) -DEXE='$(JAVA_COMMAND) $(JVM_OPTIONS) -Xdump' \
+	-Xint -jar $(CMDLINETESTER_JAR) \
+	-config $(Q)$(TEST_RESROOT)$(D)j9tests.xml$(Q) \
+	-xids all,$(PLATFORM) -plats all,$(PLATFORM) -xlist $(Q)$(TEST_RESROOT)$(D)j9tests_exclude.xml$(Q) \
+	-explainExcludes -nonZeroExitWhenError; \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
```

Comment:
ah, sorry, one more thing, please add following as this test only applies to ibm and openj9 java impl

```
<impls>
	<impl>ibm</impl>
	<impl>openj9</impl>
</impls>
```

Reply:
@llxia  this has been done! I've squashed all of the commits for convenience sake. Could you check if I implemented your change correctly?